elastic/kibana
# [Maps] use index-pattern field display name in UX

Fixes https://github.com/elastic/kibana/issues/83957

Show index-pattern field display name in UX

To test:
1) install web logs sample data set
2) Edit kibana_sample_data_logs index-pattern
3) add "mybytes" custom label for "bytes" field
4) add "mygeocoordinates" custom label for "geo.coordinates" field.
5) Ensure custom labels are used throughout maps UX

<img width="300" alt="Screen Shot 2020-12-03 at 1 22 19 PM" src="https://user-images.githubusercontent.com/373691/101084190-d4ec6600-356a-11eb-99d8-b6ae55f6ed75.png">

<img width="500" alt="Screen Shot 2020-12-03 at 1 22 45 PM" src="https://user-images.githubusercontent.com/373691/101084206-dae24700-356a-11eb-8528-171122be3357.png">

<img width="300" alt="Screen Shot 2020-12-03 at 1 24 08 PM" src="https://user-images.githubusercontent.com/373691/101084294-fd746000-356a-11eb-9c51-03161a75ef12.png">

<img width="500" alt="Screen Shot 2020-12-03 at 1 24 25 PM" src="https://user-images.githubusercontent.com/373691/101084330-0a914f00-356b-11eb-8da0-b946dadde8f8.png">
